migrations/0000_loresmith_db_schema.sql

```diff
@@ -26,7 +26,7 @@ CREATE TABLE campaign_resources (
 );
 
 -- PDF content chunks for RAG
-CREATE TABLE pdf_chunks (
+CREATE TABLE IF NOT EXISTS pdf_chunks (
```

Comment:
i think - and i think you agree - we should move away from naming things `pdf` unless they are very specific to pdf